backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -698,22 +698,21 @@ public List<Observation> convertToObservation(
                   deviceType.getSupportedDiseaseTestPerformed().stream()
                       .filter(code -> code.getSupportedDisease().equals(result.getDisease()))
                       .toList();
-              String testPerformedLoincCode =
-                  deviceTypeDiseaseEntries.stream()
-                      .findFirst()
-                      .map(DeviceTypeDisease::getTestPerformedLoincCode)
-                      .orElse(null);
+              var deviceTypeDisease =
```

Comment:
nit: can we use `DeviceTypeDisease` here instead of `var`?

Reply:
+1 to this! think there are couple places where we can be more explicit to how we're typing things / not use `var`